src/App.vue

```diff
@@ -28,11 +28,14 @@ export default {
     }
   },
   mounted() {
-    // TODO: Enable this when we start fetching data from the API
-    // this.$store.dispatch("App/setLoading", true);
-    // setTimeout(() => {
-    //   this.$store.dispatch("App/setLoading", false);
-    // }, 5000);
+    this.$store.dispatch("App/setLoading", true);
+
+    this.$store.dispatch("About/getDevelopers");
+    this.$store.dispatch("About/getRepositories");
+
+    setTimeout(() => {
+      this.$store.dispatch("App/setLoading", false);
+    }, 5000);
```

Comment:
I think we should remove the loading screen once the actions finish, not after a fixed period of time. We can use a `Promise.all` and start returning a promise in both actions.
```suggestion
    await Promise.all([
      this.$store.dispatch("About/getDevelopers"),
      this.$store.dispatch("About/getRepositories")
    ])
    this.$store.dispatch("App/setLoading", false);
```

Reply:
Done, though the linter didn't allowed to be to style the promise all that way...